cdap-ui/app/features/admin/controllers/NamespaceSettingsController.js

```diff
@@ -0,0 +1,51 @@
+angular.module(PKG.name + '.feature.admin')
+  .controller('AdminNamespaceSettingController', function ($scope, MyDataSource, $state, $alert, $timeout, myNamespace) {
+
+    var dataSrc = new MyDataSource($scope);
+    $scope.loading = false;
+
+    dataSrc.request({
+      _cdapPath: '/namespaces/' + $state.params.nsadmin
+    })
+    .then(function (res) {
+      $scope.description = res.description;
+    });
+
+    $scope.save = function() {
+
+      dataSrc.request({
+        _cdapPath: '/namespaces/' + $state.params.nsadmin + '/properties',
+        method: 'PUT',
+        body: {
+          'description': $scope.description
+        }
+      })
+      .then(function () {
+        $alert({
+          type: 'success',
+          content: 'Namespace have successfully updated'
```

Comment:
This message seems grammatically incorrect.
Either `Namespace` should have a `s` at the end, or the `have` should be replaced by a `has`.